app/assets/stylesheets/custom.scss

```diff
@@ -65,4 +65,8 @@ a.navbar-brand{
 .adress-font {
   font-size: 14px;
   color: #999999;
+}
+
+.gray-cell {

```

Comment:
May we use B4 class?
https://getbootstrap.com/docs/4.0/content/tables/#contextual-classes

Reply:
Good idea!